src/components/item-categories/index.js

```diff
@@ -1,10 +1,31 @@
 import React from 'react';
-import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Badge } from 'react-rainbow-components';
 
-const Styled = styled.div``;
+const ItemCategories = props => {
+    const { className, style, categories } = props;
 
-const ItemCategories = () => {
-    return <Styled>ItemCategories</Styled>;
+    if (categories) {
```

Comment:
Here I think that we should also check for categories.length because when it is an empty array it will return an empty div